### PR TITLE
Fix admin url include 

### PR DESCRIPTION
**Pull request by pamella**
## Description

While using the 1.0.0 version on a project, an error was raised due to the usage of the `namespace` argument, which isn't necessary.

Relates to #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ![PyPI - Version](https://img.shields.io/pypi/v/django-celerybeat-status)
 ![Supported Python Versions](https://img.shields.io/pypi/pyversions/django-celerybeat-status.svg)
+![Supported Django Versions](https://img.shields.io/pypi/frameworkversions/django/django-celerybeat-status.svg)
```
Comment (pamella) [inline README.md:5]: Drive-by addition.

File: README.md [hunk 2]
```diff
@@ -31,7 +32,8 @@ from django.urls import include, path
 
 urlpatterns = [
   # other urls...
-  path("admin/statuscheck/", include("celerybeat_status.urls", namespace="celerybeat_status")),
+  path("admin/statuscheck/", include("celerybeat_status.urls")),  # celerybeat_status admin
```
Comment (pamella) [inline README.md:35]: No need to use `namespace`. I also updated the test app.

**Review by fjsj — APPROVED** [review]
